src/main/java/com/aws/greengrass/deployment/DeviceConfiguration.java

```diff
@@ -355,6 +360,17 @@ public Topic getRootCAFilePath() {
                 .addValidator(deTildeValidator);
     }
 
+    /**
+     * Get JVM options. Default to current JVM options input.
+     *
+     * @return JVM options config topic
+     */
+    public Topic getJvmOptions() {
+        String jvmOptions = ManagementFactory.getRuntimeMXBean().getInputArguments().stream().sorted()
+                .filter(s -> !s.startsWith(JVM_OPTION_ROOT_PATH)).collect(Collectors.joining(" "));
+        return getTopic(DEVICE_PARAM_JVM_OPTIONS).dflt(jvmOptions);
```

Comment:
Should we remove the default for the `jvmOptions` config from the Nucleus recipe now that the default it evaluated here?

Reply:
Do you mean this should be removed instead of default to empty string? https://code.amazon.com/packages/GreengrassV2ComponentsReleaseCDK/blobs/47911fb22576071ee8ec09c7bf0eef3ef421ae35/--/lib/components/greengrass-nucleus.ts#L68

Reply:
I'm keep it in mind when updating the cdk